docs/troubleshooting.md

```diff
@@ -0,0 +1,15 @@
+# Troubleshooting
+
+## Missing `canvas` on macOS
+
+This issue might occur in a Silicon M1 chip:
+
+```text
+Cannot find module '../build/Release/canvas.node'
+```
+
+Solution:
+
+```sh
+brew install pkg-config cairo pango libpng jpeg giflib librsvg
+```
```

Comment:
i think after running this command, again installing the npm packages solves an issue because we have to install `canvas` package again
```suggestion
```sh
brew install pkg-config cairo pango libpng jpeg giflib librsvg
pnpm install --force
```
```

Reply:
Awesome, thanks